UnitTests/Views/ScrollViewTests.cs

```diff
@@ -602,16 +602,17 @@ public void Clear_Window_Inside_ScrollView ()
 00000000000000000000000", attributes);
 		}
 
-
 		[Fact, AutoInitShutdown]
 		public void DrawTextFormatter_Respects_The_Clip_Bounds ()
 		{
-			var view = new View (new Rect (0, 0, 20, 20));
-			view.Add (new Label ("0123456789abcdefghij"));
-			view.Add (new Label (0, 1, "1\n2\n3\n4\n5\n6\n7\n8\n9\n0"));
+			var rule = "0123456789";
```

Comment:
Can you use the new `Ruler` class here instead?

Reply:
Also, please remove the "Test" from the below so that as we evolve the look & feel of "Title" we don't have to update unit tests.

Reply:
> Can you use the new `Ruler` class here instead?

In a scroll view isn't very easy to use it, because it will force to redraw on every iteration. Fill the label text only once is much more easy.